packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -184,7 +189,8 @@ class Auth {
             idp_access_token: this.get('idp_access_token'),
             refresh_token: this.get('refresh_token_registered') || this.get('refresh_token_guest'),
             token_type: this.get('token_type'),
-            usid: this.get('usid')
+            usid: this.get('usid'),
+            auth_type: this.get('auth_type')
```

Comment:
The return type of this function is `ShopperLoginTypes.TokenResponse`, you'd probably need to change that.